applications_tests/gls_navier_stokes_2d/heat_transfer_laser.output

```diff
@@ -0,0 +1,47 @@
+Running on 1 MPI rank(s)...
+   Number of active cells:       1024
+   Number of degrees of freedom: 3267
+   Volume of triangulation:      0.01
+   Number of thermal degrees of freedom: 1089
+
+*****************************************************************
+Transient iteration : 1        Time : 0.01     Time step : 0.01     CFL : 0       
+*****************************************************************
+L2 error velocity : 0
+L2 error temperature : 150.231
+
+*****************************************************************
+Transient iteration : 2        Time : 0.02     Time step : 0.01     CFL : 0       
+*****************************************************************
+L2 error velocity : 0
+L2 error temperature : 278.378
+
+*****************************************************************
+Transient iteration : 3        Time : 0.03     Time step : 0.01     CFL : 0       
+*****************************************************************
+L2 error velocity : 0
+L2 error temperature : 378.601
+
+*****************************************************************
+Transient iteration : 4        Time : 0.04     Time step : 0.01     CFL : 0       
+*****************************************************************
+L2 error velocity : 0
+L2 error temperature : 287.622
+
+*****************************************************************
+Transient iteration : 5        Time : 0.05     Time step : 0.01     CFL : 0       
+*****************************************************************
+L2 error velocity : 0
+L2 error temperature : 245.927
+ time  error_velocity 
+0.0100 0.000000e+00   
+0.0200 0.000000e+00   
+0.0300 0.000000e+00   
+0.0400 0.000000e+00   
+0.0500 0.000000e+00   
+cells error_temperature 
+1024  1.502310e+02      
+1024  2.783781e+02      
+1024  3.786006e+02      
+1024  2.876222e+02      
```

Comment:
I guess this is temporary until you have the post-processing for max temperature right?

Reply:
Right, I will modify the output of this test after adding the post-processing.